build-logic/publishing/src/main/kotlin/build-logic.publish-to-central.gradle.kts

```diff
@@ -61,27 +59,11 @@ publishing {
             }
         }
     }
-}
-
-val createReleaseBundle by tasks.registering(Sync::class) {
-    description = "This task should be used by github actions to create release artifacts along with a slsa attestation"
-    val releaseDir = layout.buildDirectory.dir("release")
-    outputs.dir(releaseDir)
-
-    into(releaseDir)
-    rename("pom-default.xml", "${project.name}-${project.version}.pom")
-    rename("module.json", "${project.name}-${project.version}.module")
-}
-
-publishing {
-    publications.configureEach {
-        (this as PublicationInternal<*>).allPublishableArtifacts {
-            val publicationArtifact = this
-            createReleaseBundle.configure {
-                dependsOn(publicationArtifact)
-                from(publicationArtifact.file)
-            }
+    repositories {
+        maven {
+            name = "sonatype"
+            url = uri("https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/")
+            credentials(PasswordCredentials::class)
         }
     }
 }
```

Comment:
It's beyond the scope of that PR, but you might consider automatically publishing SNAPSHOT to Maven Central on every build from the default branch* (unless I miss and it is already being done).

* - some logic to build the version would be required.

Reply:
that's fine, I think we can do that in the future

Reply:
Reported as https://github.com/sigstore/sigstore-java/issues/620

(but I can't "resolve" this thread)